launchable/commands/subset.py

```diff
@@ -493,7 +516,15 @@ def run(self):
                     timeout = (5, 300)
                     payload = self.get_payload(session_id, target, duration, test_runner)
 
-                    res = client.request("post", "subset", timeout=timeout, payload=payload, compress=True)
+                    if is_non_blocking:
+                        # Create a new process for requesting a subset.
+                        process = Process(target=subset_request, args=(client, timeout, payload))
+                        process.start()
+                        click.echo("The subset was requested in non-blocking mode.")
```

Comment:
As far as I remember, click.echo outputs as stdout. It means this message ("The subset was requested in non-blocking mode.") will be included as subset out. So you should print this message as stderr. ( I know it's a little tricky 😅)

Reply:
Oh, thank you for telling me that.

Reply:
[f4cbddd](https://github.com/launchableinc/cli/pull/963/commits/f4cbddd25340e02a6feb3fac0d5d531f79074e48)